multiapps-controller-process/src/test/java/org/cloudfoundry/multiapps/controller/process/flowable/ProcessActonRegistryTest.java

```diff
@@ -0,0 +1,80 @@
+package org.cloudfoundry.multiapps.controller.process.flowable;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.text.MessageFormat;
+import java.util.List;
+
+import org.cloudfoundry.multiapps.controller.process.Messages;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ProcessActonRegistryTest {
+
+    private ProcessActionRegistry processActionRegistry;
+
+    @BeforeEach
+    void setUp() {
+        processActionRegistry = new ProcessActionRegistry(getProcessActions());
+    }
+
+    @Test
+    void testGetStartProcessAction() {
```

Comment:
This test is identical to `testGetAbortProcessAction`. Either change or remove it